Modules/Filtering/LabelMap/include/itkLabelMap.hxx

```diff
@@ -183,39 +175,25 @@ template <typename TLabelObject>
 auto
 LabelMap<TLabelObject>::GetNthLabelObject(const LabelMap::SizeValueType & pos) -> LabelObjectType *
 {
-  SizeValueType i = 0;
-
-  for (auto it = m_LabelObjectContainer.begin(); it != m_LabelObjectContainer.end(); ++it)
+  if (const auto numberOfLabelObjects = m_LabelObjectContainer.size(); numberOfLabelObjects <= pos)
   {
-    if (i == pos)
-    {
-      return it->second;
-    }
-    ++i;
+    itkExceptionMacro("Can't access to label object at position "
+                      << pos << ". The label map has only " << numberOfLabelObjects << " label objects registered.");
   }
-  itkExceptionMacro("Can't access to label object at position " << pos << ". The label map has only "
-                                                                << this->GetNumberOfLabelObjects()
-                                                                << " label objects registered.");
+  return std::next(m_LabelObjectContainer.cbegin(); pos);
```

Comment:
typo

Reply:
@issakomi Thanks but do you mean the same type that @dzenanz mentioned?

Reply:
I mean `;` after ` cbegin()`

Reply:
Thanks @issakomi, also addressed by the force-push that I just did!